getting-started/env.md

```diff
@@ -12,50 +12,81 @@ This raises a number of questions:
+![basic_railway_elements.drawio.png](../assets/images/basic_railway_elements.drawio.png)
 
-![Imgur](https://i.imgur.com/Q72tAI8.png)
-*8 unique cells enable us to implement any realworld railway network in the flatland env*
+*10 basic cells modulo rotation enable us to implement any realworld railway network in the flatland env*
+This gives a set of 30 valid transitions in total (see `#` giving number of rotations).
+
+Agents can only travel in the direction they are currently facing. Hence, the permitted transitions for any given agent depend both on its position and on its
+direction. Transition maps define the railway network in the flatland world. One can implement any real world railway network within the Flatland environment by
+manupulating the transition maps of cells.
 
-Agents can only travel in the direction they are currently facing. Hence, the permitted transitions for any given agent depend both on its position and on its direction. Transition maps define the railway network in the flatland world. One can implement any real world railway network within the Flatland environment by manupulating the transition maps of cells.
- 
 For more information on transtion maps checkout [environment information](../environment/environment_information)!
 
 
 ↔️ Actions
 ---
 
-The trains in Flatland have strongly limited movements, as you would expect from a railway simulation. This means that only a few actions are valid in most cases. 
+The trains in Flatland have strongly limited movements, as you would expect from a railway simulation. This means that only a few actions are valid in most
+cases.
 
 Here are the possible actions:
 
-- **`DO_NOTHING`**:  If the agent is already moving, it continues moving. If it is stopped, it stays stopped. Special case: if the agent is at a dead-end, this action will result in the train turning around.
-- **`MOVE_LEFT`**: This action is only valid at cells where the agent can change direction towards the left. If chosen, the left transition and a rotation of the agent orientation to the left is executed. If the agent is stopped, this action will cause it to start moving in any cell where forward or left is allowed!
+- **`DO_NOTHING`**:  If the agent is already moving, it continues moving. If it is stopped, it stays stopped. Special case: if the agent is at a dead-end, this
+  action will result in the train turning around.
+- **`MOVE_LEFT`**: This action is only valid at cells where the agent can change direction towards the left. If chosen, the left transition and a rotation of
+  the agent orientation to the left is executed. If the agent is stopped, this action will cause it to start moving in any cell where forward or left is
+  allowed!
 - **`MOVE_FORWARD`**: The agent will move forward. This action will start the agent when stopped. At switches, this will chose the forward direction.
```

Comment:
```suggestion
- **`MOVE_FORWARD`**: The agent will move forward. This action will start the agent when stopped. At switches, this will choose the forward direction.
```
https://www.merriam-webster.com/dictionary/choose